source/preprocessor.tex

```diff
@@ -1783,19 +1793,15 @@
 Otherwise, the original spelling of each preprocessing token in the
 stringizing argument is retained in the character string literal,
 except for special handling for producing the spelling of
-\grammarterm{string-literal}s and \grammarterm{character-literal}s:
-a
-\tcode{\textbackslash}
-character is inserted before each
-\tcode{"}
-and
-\tcode{\textbackslash}
-character of a \grammarterm{character-literal} or \grammarterm{string-literal}
-(including the delimiting
-\tcode{"}
-characters).
+\grammarterm{character-literal}s and \grammarterm{string-literal}s
+(including the delimiting \unicode{0022}{quotation mark} (\tcode{"}))
+contained within the preprocessing token:
+a \unicode{005c}{reverse solidus} character (\tcode{\textbackslash})
+is inserted before each \unicode{0022}{quotation mark} and
+\unicode{005c}{reverse solidus} character and each new-line character is
+replaced by the two-character sequence \tcode{\textbackslash n}.
 If the replacement that results is not a valid character string literal,
-the behavior is undefined. The character string literal corresponding to
+the program is ill-formed. The character string literal corresponding to
```

Comment:
Would be nice to put the next sentence on a separate line while we're at it (semantic line breaks).

Reply:
Re-ordered the two sentences, and then broke the large paragraph into several smaller ones to improve readability.